docker-compose.full-stack.yml

```diff
@@ -121,3 +121,9 @@ services:
       interval: 10s
       timeout: 5s
       retries: 5
+  # Hapi container
+  hapi-server:
+    image: hapiproject/hapi:latest
+    restart: always
+    ports:
+      - "8080:8080"
```

Comment:
This config shouldn't make its way into main, but is useful for the pre-commit POC